README.md

```diff
@@ -129,4 +129,9 @@ Now when you navigate to your localhost (e.g. `http://localhost:3000/`) that ser
 
 In the form above, users can enter a URL to check, as well as select a model of choice.
 
-To stop the application, use `docker-compose down`.
\ No newline at end of file
+To stop the application, use `docker-compose down`.
+
+# Prometheus and Grafana
+
+To run Prometheus locally without helm, you need to make sure Prometheus is installed, then run prometheus.exe --config.file=prometheus.yml.
+To run Grafana locally, just download it and it will continually run on your localhost on port 3000 (use a browser to go to 127.0.0.1:3000). Then configure a data source to the port Prometheus is running (by default, localhost:9093) and importing the dashboard JSON located in the path grafana-dashboards/dashbard.json.
```

Comment:
```suggestion
To run Grafana locally, just download it and it will continually run on your localhost on port 3000 (use a browser to go to 127.0.0.1:3000). Then configure a data source to the port Prometheus is running (by default, localhost:9093) and importing the dashboard JSON located in the path *grafana-dashboards/dashbard.json*.
```